src/tests/test-profile/meson.build

```diff
@@ -0,0 +1,16 @@
+if build_standalone
+  gzip = find_program('gzip')
+  foreach input_file : [
+    'thinkpad-p1-no-iommu.json',
+    'thinkpad-p1-iommu.json',
+  ]
+    custom_target(input_file,
+      input: input_file,
+      output: '@0@.gz'.format(input_file),
+      capture: true,
+      command: [gzip, '--keep', '--stdout', '@INPUT@'],
+      install: true,
```

Comment:
I'm still not convinced we want this to be installed by default, especially as we add more data for other systems and it grows.

Reply:
Maybe one of them? I wanted something I can reference in the commit and maybe the docs.